pynucastro/networks/tests/test_derived_network.py

```diff
@@ -67,11 +67,7 @@ def fe52__p_mn51__derived(tf):
     rate += np.exp(  {:.14f} + -85.6326403498911*tf.T9i + -36.1825*tf.T913i + 0.873042*tf.T913
                   + -2.89731*tf.T9 + 0.364394*tf.T953 + 0.833333*tf.lnT9)
```

Comment:
I don't understand this -- why is there a `{:14f}` here?

Reply:
That is a placeholder for `[61.72798916565748, 61.72798916565747]`

Reply:
I used both values, because depending on the numpy version it generates one number or the another.